scripts/create-release-branch.js

```diff
@@ -89,10 +100,7 @@ shell.exec('git add lerna.json');
 shell.exec(`git commit -m "chore: update to version ${nextVersion}"`);
 
 // Push version update commits to develop
-shell.exec(`git push origin develop`);
-
-// Create the new release branch and switch to it
-shell.exec(`git checkout -b ${releaseBranchName}`);
+// shell.exec(`git push origin develop`);
```

Comment:
nit: delete.  no dead code.

Reply:
I'm on the fence about this one. This would eliminate the cherry-pick process to add one of changes, but the downside is that we are no longer tied to the week. I think the last time we discussed the majority seemed to agree to this, so let me just delete it.